src/app/berkeley_migration/precomputed_block.ml

```diff
@@ -220,7 +220,7 @@ let concrete_fetch_batch ~logger ~bucket ~network targets =
       in
       don't_wait_for
         (let rec progress_loop () =
-           let%bind existing = list_directory ~network in
+           let%bind existing = list_directory ~network ~path:local_path in
```

Comment:
Why don't we make `--precomputed-local-path` an optional argument and if it's provided, attempt to create a directory if it doesn't exist and instead of listing precomputed blocks here just remove this directory?

I remember @nholland94 and I were discussing something along these lines

Reply:
Actually this is good point. Maybe this parameter should be optional and by default set to '.'. However, I don't understand why instead of listing directory we should remove it if it does not exist ?